docs/articles_en/documentation/openvino-ir-format/operation-sets/operation-specs/signals/istft-16.rst

```diff
@@ -0,0 +1,215 @@
+  * ``default_signal_length = (frames - 1) * frame_step + frame_size`` for ``center == false`` 
+
+If the ``signal_length`` input is provided, the number of output values will be adjusted accordingly. 
+  * If ``signal_length > default_signal_length`` the output is padded with zeros at the end.
+  * If ``signal_length < default_signal_length`` any additional generated samples are cut to the ``signal_length`` size.
+
+The ``window_length`` can not be larger than ``frame_size``, but if smaller the window values will be padded with zeros on the left and right side. The size of the left padding is calculated as ``(frame_size - window_length) // 2``, then right padding size is filled to match the ``frame_size``.  
+
+**Attributes**:
+
+* *center*
+
+  * **Description**: Flag that indicates whether padding has been applied to the original signal. It affects output shape, if the ``signal_length`` input is not provided.
+  * **Range of values**:
+
+    * ``false`` - padding has not been applied, default signal length is calculated as ``(frames - 1) * frame_step + frame_size``
+    * ``true`` - padding has been applied, default signal length is calculated as ``(frames - 1) * frame_step``
+  * **Type**: ``boolean``
+  * **Required**: *yes*
+
+* *normalized*
+
+  * **Description**: Flag that indicates whether the input has been normalized. It is needed to correctly restore the signal and denormalize the output. Output of the STFT is divided by ``sqrt(frame_size)``, when normalized.
+  * **Range of values**:
+
+    * ``false`` - input has not been normalized
+    * ``true`` - input has been normalized
+  * **Type**: ``boolean``
+  * **Required**: *yes*
+
+
+**Inputs**
+
+* **1**: ``data`` - Tensor of type *T*, the ISTFT data input (compatible with a result of STFT operation). **Required.**
+  * The data input shape can be 3D ``[fft_results, frames, 2]`` or 4D ``[batch, fft_results, frames, 2]``.
+* **2**: ``window`` - Tensor of type *T* and 1D shape ``[window_length]``, specifying the window values applied to restore the signal. The ``window_length`` is required to be equal or smaller than ``frame_size``, if smaller the window will be padded with zeros on the left and right sides. **Required.**
+* **3**: ``frame_size`` - Scalar tensor of type *T_INT* describing the size of a single frame of the signal to be provided as input to FFT. **Required.**
+* **4**: ``frame_step`` - Scalar tensor of type *T_INT* describing the distance (number of samples) between successive frames. **Required.**
+* **5**: ``signal_length`` - Scalar or single element 1D tensor of type *T_INT* describing the desired length of the output signal, if not provided it's calculated accordingly to the rules presented in the detailed description above. **Optional.**
+
+
+**Outputs**
+* **1**: ``signal`` - Tensor of type *T* and 1D shape ``[signal_length]`` or 2D shape ``[batch, signal_length]`` with a real valued signal data. **Required.**
```

Comment:
Docs for STFT use term `signal_size` instead `signal_length`. I think it is quite clear but maybe it would be worth to align those names between ops?

Reply:
Sure, it makes sense to use the same term. But I would like to avoid making changes in other files within this PR, and apply such polishing separately if needed.